### PR TITLE
css two column layout | Humberto Navarro

### PR DESCRIPTION
**Pull request by humbertovnavarro**

### CODE REVIEW

**Comment by humbertovnavarro** [comment]
image

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Humberto! I can't see your entire output in your screenshot, please submit another image so that I can see the entire output

**Comment by humbertovnavarro** [comment]
image

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Humberto! The two columns have too much space between each other and the rows above and below them as compared to the example. Please revise this to match the desired output

**Comment by humbertovnavarro** [comment]
image

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  border: 2px solid #d14c45;
+  font-weight:600;
+  background-color: transparent;
+  color: #d14c45;
+  padding-left: auto;
+  padding-right: auto;
+}
+
+button:hover {
+  background-color :#d14c45;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:83]: The space between your property and the `:` here is messing up your syntax

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here! One minor note of housekeeping:

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (codingcodymiller) [reply css-two-column-layout/styles.css:89]: You misunderstood, the selector was fine, I mean the space after `background-color`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Great work on this!!!

**Pull request merged**